pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -145,7 +145,7 @@ func (pc *PackageControllerClient) GetCuratedPackagesRegistries() (sourceRegistr
 			defaultImageRegistry = gatedOCINamespace
 		}
 	} else {
-		defaultImageRegistry = strings.ReplaceAll(constants.DefaultCuratedPackagesRegistryRegex, "*", pc.eksaRegion)
+		defaultImageRegistry = strings.ReplaceAll(defaultImageRegistry, "us-west-2", pc.eksaRegion)
```

Comment:
Is there region replication yet on the dev images?

Reply:
@jonahjon would know, but either way we should support it and have replication to at least one region.